eea/dexterity/indicators/contentrules/version.py

```diff
@@ -128,9 +150,25 @@ def __call__(self):
 
         notify(ObjectClonedEvent(obj))
 
-        pr = getToolByName(obj, 'portal_repository')
+        pr = getToolByName(obj, "portal_repository")
         pr.save(obj=obj, comment=change_note)
 
+        # CHANGE URL OF FIGURES TO THE NEW DRAFT VERSION
+        for block_data in visit_blocks(obj, obj.blocks):
+            if block_data.get("@type") == "dataFigure" and "url" in block_data:
```

Comment:
@dobri1408 As we migrated to embed_content in PR #44 and #53 we should also update this PR.